.github/workflows/RunnerFastCI.yml

```diff
@@ -46,37 +42,29 @@ jobs:
     - name: Compile NHVHPC GPU NCCL
       run: |
         ls -ltr ~/.bashrc
-        source ~/.bashrc
-        source ~/.profile_github
-        module load CMake 
+        source /etc/profile.d/lmod.sh
+        source ~/.bash_aliases
         module load nvhpc
         export FC=mpif90
         cmake -S . -B build -DBUILD_TESTING=ON -DBUILD_TARGET=gpu -DENABLE_NCCL=yes
         cmake --build build -j 4
         cmake --install build 
         ctest --test-dir build 
-        cmake -S . -B build -DNX=128 -DNY=128 -DNX=128
-        cmake --build build -j 4
-        cmake --install build 
-        ctest --test-dir build
         rm -rf build  
     
     # Configure-Build-Run-Run on 4 cores 
     - name: Compile GNU_Dev openMPI FFTW_F03 ADIOS2 CPU
       run: |
         ls -ltr ~/.bashrc
-        source ~/.bashrc
-        source ~/.profile_github
-        module load CMake 
+        source /etc/profile.d/lmod.sh
+        source ~/.bash_aliases
         module load foss
         export FC=mpif90
         export CC=mpicc
         export CXX=mpicxx
-        cmake -S . -B build -DCMAKE_BUILD_TYPE=Dev -DBUILD_TESTING=ON -DIO_BACKEND=adios2 -Dadios2_DIR=~/GIT_Work/ADIOS2/build_install_gcc12_cpu/opt/lib/cmake/adios2 -DFFT_Choice=fftw_f03
-        cmake --build build -j 4
-        cmake --install build 
-        ctest --test-dir build 
-        cmake -S . -B build -DNX=128 -DNY=128 -DNX=128 -DMPIEXEC_MAX_NUMPROCS=4
+        #cmake -S . -B build -DCMAKE_BUILD_TYPE=Dev -DBUILD_TESTING=ON -DIO_BACKEND=adios2 -Dadios2_DIR=~/GIT_Work/ADIOS2/build_install_gcc12_cpu/opt/lib/cmake/adios2 -DFFT_Choice=fftw_f03
```

Comment:
Should we remove this line to clean up the config or is it being kept for later CI updates?

Reply:
All commented code has been removed and all functionalities have been restored